src/Microsoft.Identity.Web.TokenCache/Microsoft.Identity.Web.TokenCache.csproj

```
@@ -11,7 +11,7 @@
     <PackageReference Include="Microsoft.Identity.Client" Version="$(MicrosoftIdentityClientVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net6.0' Or '$(TargetFramework)' == 'net7.0' or '$(TargetFramework)' == 'net8.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net6.0' Or '$(TargetFramework)' == 'net7.0' Or '$(TargetFramework)' == 'net8.0' Or '$(TargetFramework)' == 'net9.0'">
```

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFrameworkIdentifier)' == 'NET6_0_OR_GREATER'">
```
Similar to the other one, would this be better?

---

Yeap, Thanks Westin!